configs/README.md

```diff
@@ -0,0 +1,35 @@
+# Workflow Dispatch Configs
+
+This directory contains config files used for workflow dispatch.
+
+| name | description |
+| --- | --- |
+| go | repositories containing Go code |
+| testing | repositories used for testing unified CI workflows |
+
+## Adding new repository to existing config file
+
+To add a new repository to an existing config file, add a new JSON object to the `repositories` key in the file.
+
+_**IMPORTANT**: Please remember to keep `repositories` sorted alphabetically by `target` value._
+
+*JSON object example:*
+```
+{ "target": "NEW REPOSITORY NAME" }
+```
+
+After the PR with your change is merged, a copy workflow that runs in this repository will copy files(as defined by the `defaults.files` key of the config file or an overriden `files` key of your newly added object) to your repository.
+
+## Adding new config file
+
+When adding a new JSON config file, please follow the structure of other config files. In particular, `defaults.files` array and `repositories` array are required fields.
+
+*JSON config file example:*
+```
+{
+    "defaults": { "files": [] },
+    "repositories": []
+}
+```
+
+To customise the copy workflow further, you can add more fields to the `defaults` object. See `deploy_versioning` or `deploy_go` in [go.json](go.json#L8-L9) and how they are used in [copy-workflow.yml](../.github/workflows/copy-workflow.yml#L100-L105) for example.
```

Comment:
`deploy_versioning` is now gone.

Reply:
Not entirely. I've left this part of the `deploy_versioning` flag - https://github.com/protocol/.github/blob/a71f87a0d506dfde49763755e91f1eb8732828cb/.github/actions/copy-workflow-versioning/action.yml

Other options here are:
1. move the steps from `copy-workflow-versioning` to https://github.com/protocol/.github/blob/a71f87a0d506dfde49763755e91f1eb8732828cb/.github/actions/copy-workflow-go/action.yml
2. execute `copy-workflow-versioning` only if ```".github/workflows/releaser.yml", ".github/workflows/release-check.yml"``` are being copied

Personally, I'd pick between what's currently implemented in this PR(explicit `deploy_versioning` flag) or option 2. Either of them would allow to opt-out or opt-in to versioning from other configurations. I wouldn't want to go for 1 because versioning setup, as it is now, is not go specific and if we could keep it this way it'd be great.

Reply:
> I wouldn't want to go for 1 because versioning setup, as it is now, is not go specific and if we could keep it this way it'd be great.

That's true for `tagpush` and `release`, but not for `release-check`. That one is very much Go specific.
How do we deal with that?

Reply:
Oh yeah, you're right about the `release-check`. Actually, now that I think about it, it's quite unlikely we'll be able to reuse `release` either. That's because both `js` and `rust`(which are the two most likely next candidates) have more complicated release processes(`npm`, `crate`) than just publishing git tags. So we'll have to create new workflows for them anyway.

All this makes me feel better about just moving the `copy-workflow-versioning` action content to `copy-workflow-go` action. Here's a PR for this if we want to go with it: https://github.com/protocol/.github/pull/256

One last tiny reservation I have is the ability to opt-out of versioning(which `go-ds-crdt` wants to do 😔). In the current implementation that would be possible by changing the repo object to:
```
{ 
  "target": "ipfs/go-ds-crdt", 
  "files":  [
      ".github/workflows/automerge.yml",
      ".github/workflows/go-test.yml",
      ".github/workflows/go-check.yml"
    ],
    "deploy_versioning": false
}
```

Reply:
@marten-seemann What do you think? Personally, I think I still feel the best about the option that is currently implemented here.

Reply:
Not sure what to do with go-ds-crdt (and if we as an org are ok with a poorly tested repo like that), but given that they deactivated the workflow altogether, we probably shouldn't pay too much attention to them.

Reply:
Which way do you think we should go here then? I'm happy either way.